### PR TITLE
[flutter_tools] Generate correct entrypoint module name

### PR DESCRIPTION
**Pull request by jonahwilliams**
## Description

Ensure that the bootstrap module is requested as `packages/package-name/path` instead of an absolute file path. Removes windows file-path workarounds from module mapping.

### CODE REVIEW

File: packages/flutter_tools/lib/src/web/devfs_web.dart [hunk 1]
```diff
@@ -347,17 +341,20 @@ class WebDevFS implements DevFS {
         'web',
         'dart_stack_trace_mapper.js',
       ));
+      final String entrypoint = PackageUriMapper(mainPath, '.packages', null, null)
```
Comment (jakemac53) [inline packages/flutter_tools/lib/src/web/devfs_web.dart]: Is this always a path under `lib`? What does `PackageUriMapper` do for files not under lib?

Comment (jonahwilliams) [reply packages/flutter_tools/lib/src/web/devfs_web.dart]: If it's not under lib, it returns null - so I'm falling back to mainPath. That ... kinda works, what does the frontend_server produce in this case?

Comment (jakemac53) [reply packages/flutter_tools/lib/src/web/devfs_web.dart]: Frontend_server will fall back to either the multi-root or absolute file uri (multi-root takes precedence).

The problem is down below we do `/packages/$entrypoint.lib.js` which won't work with the absolute or multi root uris.

Comment (jonahwilliams) [reply packages/flutter_tools/lib/src/web/devfs_web.dart]: Ahh yeah, I'd need to check for the whole thing being null. I'll update with some tests

**Review by jakemac53 — APPROVED** [review]

**Pull request merged**